web/src/components/TimeSlider.tsx

```diff
@@ -88,7 +93,10 @@ export function TimeSliderBasic({
       max={numberOfEntries}
       step={1}
       value={selectedIndex && selectedIndex > 0 ? [selectedIndex] : [0]}
-      onValueChange={(value) => onChange(value[0])}
+      onValueChange={(value) => {
+        onChange(value[0]);
+        trackTimeSliderEvent(value[0]);
```

Comment:
As far as I can tell `value[0]` is just a number, e.g. for hourly its 0-24 and monthly 0-12, but how do we know if that number corresponds to hourly, daily, monthly or yearly? Or are we not interested in that?

Reply:
Good point, I think we should include the time aggregate too. I'll update the PR to include it